src/Microsoft.ML.PipelineInference/TextFileContents.cs

```diff
@@ -114,7 +114,7 @@ private static bool TryParseFile(IChannel ch, TextLoader.Arguments args, IMultiS
             try
             {
                 // No need to provide information from unsuccessful loader, so we create temporary environment and get information from it in case of success
-                using (var loaderEnv = new ConsoleEnvironment(0, true))
+                using (var loaderEnv = new ConsoleEnvironment(0, verbose: true))
```

Comment:
Why are we still using ConsoleEnvironment here, but removing it from other places?

Reply:
The .AddListener below. While I've tried to limit the usage of console environment, things that act directly on the host environment base and its own mechanisms will still continue to need to use either this, local environment, or some other host environment base derived class, which MLContext (deliberately) is not.

---
In reply to: [234251385](https://github.com/dotnet/machinelearning/pull/1626#discussion_r234251385) [](ancestors = 234251385)